components/dashboard/src/projects/new-project/NewProjectRepoList.tsx

```diff
@@ -45,12 +45,9 @@ export const NewProjectRepoList: FC<Props> = ({ filteredRepos, noReposAvailable,
                                         </Button>
                                     ) : (
                                         <p className="text-gray-500 font-medium">
-                                            <a rel="noopener" className="gp-link" href={userLink(r)}>
-                                                @{r.inUse.userName}
-                                            </a>{" "}
-                                            already
```

Comment:
**praise**: Thanks for removing this. Besides the buggy behavior of this feature, this won't be needed with Projects v2, as we we'll most probably allow multiple imports of the same repository, etc.